src/main/scala/mesosphere/marathon/event/http/HttpEventStreamActor.scala

```diff
@@ -16,49 +22,117 @@ trait HttpEventStreamHandle {
   def close(): Unit
 }
 
+class HttpEventStreamActorMetrics @Inject() (metrics: Metrics) {
+  val numberOfStreams: AtomicIntGauge =
+    metrics.gauge(metrics.name("api", getClass, "number-of-streams"), new AtomicIntGauge)
```

Comment:
What about moving `"service"` and `"api"` as constants to an object `mesosphere.marathon.MetricPrefixes`?

I would not like to introduce an "api" method in Metrics since I would like to keep it general.